src/coreclr/pal/src/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ elseif (CLR_CMAKE_TARGET_FREEBSD)
   include_directories(SYSTEM $ENV{ROOTFS_DIR}/usr/local/include)
 endif()
 
-if(NOT CLR_CMAKE_USE_SYSTEM_LIBUNWIND)
+if(NOT CLR_CMAKE_USE_SYSTEM_LIBUNWIND AND NOT CLR_CMAKE_TARGET_BROWSER)
```

Comment:
Would `CLR_CMAKE_TARGET_ARCH_WASM` be more appropriate here?

Reply:
We now target only browser so far, the other possible wasm targets would be the same, at least until we have a wasm codegen available.